src/linecodes.jl

```diff
@@ -36,8 +36,20 @@ module LineCodes
     end
 
     """Capacitance matrix, nF per unit length (Getter)"""
-    function Cmatrix()::Vector{Float64}
-        return Utils.get_float64_array(Lib.LineCodes_Get_Cmatrix)
+    function Cmatrix()::Matrix{Float64}
+        n = Phases()
+        if n == 0
+            cmatrix = reshape([], (n, n))
```

Comment:
Because of the return type annotation on the function this will always return `Vector{Float64}`. But I can make this change for clarity and for possibly making the return type conversion a no-op.

Reply:
Resolved in 7e1396d.